tests/python/unittest/test_gluon_data_vision.py

```diff
@@ -45,6 +45,30 @@ def test_normalize():
     assert_almost_equal(data_expected, out_nd.asnumpy())
 
 
+@with_seed()
+def test_random_resize_crop():
+    def _test_random_resize_crop_with_diff_type(dtype):
+        # test normal case
+        data_in = nd.random.uniform(0, 255, (300, 200, 3)).astype(dtype)
+        out_nd = transforms.RandomResizedCrop((100, 120))(data_in)
+        assert out_nd.shape == (120, 100, 3), out_nd.shape
+        # test 4D input
+        data_bath_in = nd.random.uniform(0, 255, (3, 300, 200, 3)).astype(dtype)
+        out_batch_nd = transforms.RandomResizedCrop((50, 70))(data_bath_in)
+        assert out_batch_nd.shape == (3, 70, 50, 3), out_nd.shape
+        # test normal case
+        data_in = nd.random.uniform(0, 255, (50, 30, 3)).astype(dtype)
+        out_nd = transforms.RandomResizedCrop((140, 100))(data_in)
+        assert out_nd.shape == (100, 140, 3), out_nd.shape
+         # test size is larger than input image 4D
+        data_in = nd.random.uniform(0, 255, (4, 50, 30, 3)).astype(dtype)
+        out_nd = transforms.RandomResizedCrop((100, 140))(data_in)
```

Comment:
Could you add test cases for float scales too? And different scales for different dimensions?